guides/security.md

```diff
@@ -0,0 +1,65 @@
+
+Since an offline computer has no connection, it is extremely hard for someone without physical access to it to access the account’s keys. However, this also makes every transaction an extremely manual process. A common practice instead is to maintain two accounts: one offline account that securely holds the majority of your assets and another online account that holds only a few assets. Most transactions can be performed with the online account and, when its funds are low, a person can manually replenish it from the offline account.
+
+You can think of this method like having both a bank vault and a cash register drawer. Most of the time the vault is closed and locked. It is only opened occasionally (and under specific procedures) to replenish a register drawer that is running low or to store excess funds from a register drawer that is overflowing. If someone attempts to rob the bank, it is extremely hard for them to get away with anything more than what was in the register drawer.
+
+
+## Require multiple authorizations or signers
+
+Sensitive accounts can be secured by requiring authorization from multiple individuals to make a transaction. Read the [multisignature guide](concepts/multi-sig.md) to learn more about how.
+
+If you require multiple signers, you should also ensure that you do not require all the possible signers to sign a transaction. If one of the signers loses the keys to their account, you will no longer be able to perform transactions if they have to sign them.
+
+
+## Ensure assets are revocable
+
+If you issue your own assets, you should usually ensure that they can be revoked using the [“authorization revocable” flag on the account](concepts/accounts.md#flags). This allows you to effectively freeze your assets in someone else’s account in case of theft or in other extenuating circumstances.
+
+
+## What if an account’s keys are compromised?
+
+Because Stellar’s security is based around public key encryption, it’s critical that an account’s secret seed is not shared. Anyone who has access to the seed effectively has control of the account. However, if someone learns your account’s seed or you accidentally share it with someone who shouldn’t know it, you can remove its ability to control the account with the following steps:
+
+1. Make a new account.
+2. Add the new account as a signer on the compromised account. (Use the [`set options` operation](concepts/list-of-operations.md#set-options).
+3. Remove the compromised key’s signing authority on the compromised account.
+4. Now the new account controls the compromised account and the compromised account’s keys are no longer able to sign transactions.
+
+(NOTE: You must notify the owners of other accounts that the compromised account has signing authority or that the keys are compromised. They need to remove the compromised account as a signer as well [and add the new account].)
```

Comment:
@bartekn:

> It all depends what's the state of the account in the ledger. If the compromised keys are already removed from signers

Yeah, I guess I’m trying to understand the optimistic case here, where the owner of C has already removed the original key as a signer. Obviously if that hasn’t happened, A is vulnerable to malicious use by a third party who has obtained the original keys to C.

> no matter what A does, network will simply reject the transaction.

Is that because I said “A requires signatures from accounts B **and** C?” So, to make sure I’m getting this, D *cannot* sign a transaction for A on behalf of C? To put it another way: does the list of signers for A denote *public keys* that can sign and NOT *accounts* that can sign? Or is it something more complicated than that?

In my head I was actually thinking “B **or** C” (I mistyped, sorry). Does that change things?